imgui-examples/examples/support_gfx/mod.rs

```diff
@@ -58,7 +57,7 @@ pub fn run<F: FnMut(&Ui) -> bool>(title: String, clear_color: [f32; 4], mut run_
     let mut mouse_state = MouseState::default();
     let mut quit = false;
 
-    loop {
+    'running: loop {
```

Comment:
May I ask you guys the implications of using this? I thought these labels were designed for nested loops. Is it necessary to have it here?

Reply:
@kyrylo A leftover from an attempted refactor that didn't work out. The label can be removed.

Reply:
Thanks! I opened a merge request addressing what you guys reported: https://github.com/Gekkio/imgui-rs/pull/145